.dockerignore

```diff
@@ -1,9 +1,16 @@
-.vagrant/
-_build/
-apps/aecuckoo/c_src
-apps/aecuckoo/priv/bin
-apps/aecuckoo/priv/lib
-apps/aecore/priv/keys
-system_test
-erl_crash.dump
-aevm_external
+*
```

Comment:
From https://docs.docker.com/engine/reference/builder/#dockerignore-file
> Finally, you may want to specify which files to include in the context, rather than which to exclude. To achieve this, specify * as the first pattern, followed by one or more ! exception patterns.

Ok.